js/components/operatorSignIn/list.tsx

```diff
@@ -39,6 +40,10 @@ export const List = ({ line }: { line: HeavyRailLine }): ReactElement => {
             <td className="fs-mask border-y md:border-x p-1 break-all">
               {si.signed_in_employee}
             </td>
+            <td className="fs-mask border-y md:border-x p-1 break-all">
+              {/* {si.radio_number} */}
+              2222
```

Comment:
This is why tests are failing.

Reply:
Related to this field being nullable from a data perspective, maybe this should gracefully display an empty/missing state if the radio number is missing?

Reply:
doing so now 👍